compute_modules/logging/common.py

```diff
@@ -27,24 +27,42 @@
     _LoggerAdapter = logging.LoggerAdapter
 
 
+LOG_FORMATTER = None
+LOG_HANDLERS = None
+
+
 # TODO: add replica ID to default log format
 DEFAULT_LOG_FORMAT = (
     "%(levelname)-8s PID: %(process_id)-6s JOB: %(job_id)-36s LOC: %(filename)s:%(lineno)d - %(message)s"
 )
 
 
+def _setup_logger(
+    handlers: Optional[Union[logging.Handler, Tuple[logging.Handler, ...]]], formatter: Optional[logging.Formatter]
```

Comment:
Might want to set defaults of `None` here if a user only wants to set a `formatter` but not `handlers` and vice-versa

```suggestion
    handlers: Optional[Union[logging.Handler, Tuple[logging.Handler, ...]]] = None, 
    formatter: Optional[logging.Formatter] = None,
```